DocumentService/Controllers/DocumentsController.cs

```diff
@@ -28,6 +30,26 @@ public DocumentsController(IDocumentRepository documentRepository, IAzureBlobSer
             this.configuration = configuration;
         }
 
+        /// <summary>
+        /// Use for testing purposes
+        /// </summary>
+        /// <param name="file">File to upload</param>
+        /// <returns></returns>
+        [HttpPost]
+        [Route("v1/documents/test")]
+        [ProducesResponseType(StatusCodes.Status201Created)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public async Task<IActionResult> UploadDocumentAsync(IFormFile file)
+        {
+            var result = await this.azureBlobService.UploadFileAsync(file, configuration.GetSection("BlobContainers")["Documents"]);
+
+            var blob = azureBlobService.GetBlob(configuration.GetSection("BlobContainers")["Documents"], result.Uri.AbsoluteUri);
+
+            var match = blob.Properties.ContentType == file.ContentType;
```

Comment:
Is there a reason we're checking for a match? shouldn't it always be the same?

Reply:
This is just for testing purposes.